selfdrive/car/chrysler/values.py

```diff
@@ -23,6 +23,8 @@ class CAR(StrEnum):
   PACIFICA_2018 = "CHRYSLER PACIFICA 2018"
   PACIFICA_2020 = "CHRYSLER PACIFICA 2020"
 
+  DURANGO = "DODGE DURANGO 2021"
```

Comment:
```suggestion
  DODGE_DURANGO = "DODGE DURANGO 2021"
```

Any platform that isn't the current brand should generally be prefixed with it (LEXUS, JEEP, RAM, etc.)

Reply:
done